spec/view_models/auction_status/expiring_view_model_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe AuctionStatus::ExpiringViewModel do
```

Comment:
When we are dealing with objects that are *not* similar in behavior, but just happen to be the same type of thing, I think namespacing is less helpful. 

So status-type view models have namespacing of `AuctionStatus` but all view models are already logically organized by living in the `view_models` dir...am I making sense?